packages/cli/src/cli.ts

```diff
@@ -41,11 +45,12 @@ const main = async () => {
       'deck source type (choices: "tsx", "jsx", "mdx", "onepage")'
     )
     .requiredOption('-n, --name [name]', 'name of presentation')
+    .option('-l, --lang [lang]', 'language code for generated HTML document')
+    .option('-p, --port [port]', 'default port for webpack dev server')
```

Comment:
Maybe the description should mention the default values?

Reply:
Good idea!